var/spack/repos/builtin/packages/nemo/package.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Nemo(Package):
+    """The "Nucleus for European Modelling of the Ocean" (NEMO) is a 
+    state-of-the-art modelling framework. It is used for research 
+    activities and forecasting services in ocean and climate sciences. 
+    NEMO is developed by a European consortium with the 
+    objective of ensuring long term reliability and sustainability."""
+
+    homepage = "https://www.nemo-ocean.eu/"
+    git = "https://forge.nemo-ocean.eu/nemo/nemo.git"
+
+    version("4.2.1", tag="v4.2.1", commit="9c0f2d511690c30aec45516361e032c5b46a94f5")
+    version("4.2.0", tag="v4.2.0", commit="5cf0c3d61b751a102fe9de9c8ffe61a7216946cc")
+
+
+    depends_on("mpi")
+    depends_on("xios@develop")
+    depends_on("perl")
+    depends_on("netcdf-c+mpi")
+    depends_on("netcdf-fortran")
+    depends_on("hdf5+mpi")
+
+    patch("fj-v4.2.0.patch", when="@4.2.0%fj")
+    patch("fj-v4.2.1.patch", when="@4.2.1%fj")
```

Comment:
These two patches are massive. Can we download them from somewhere instead? Also, in general, it is good to  add a brief comment above each `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from